src/components/DailyDataDisplay/DailyDataDisplay.js

```diff
@@ -0,0 +1,95 @@
+// Fetches data and handles user selection for daily-resolution indicators.
+// Receives an area, a model, an emissions scenario, and a list of available 
+// datasets from its parent (DataDisplay) prefiltered to daily-only, 
+// displays an indicator-selection  dropdown (VariableSelector), 
+// a climatology dropdown (ClimatologySelector), 
+// and a graph (AnnualCycleGraph).
+
+import {annualCycleDataRequest} from '../../data-services/pcex-backend.js'
+import {noGraphMessage} from '../../helpers/GraphHelpers.js'
+import DailyGraph from '../DailyGraph/DailyGraph.js'
+import VariableSelector from '../selectors/VariableSelector.js';
+import ClimatologySelector from '../selectors/ClimatologySelector.js';
+import React, {useState, useEffect} from 'react';
+import _ from 'lodash';
+
+function DailyDataDisplay({region, rasterMetadata, model, emission}){
+
+  const [dailyTimeSeries, setDailyTimeSeries] = useState(null);
+  const [variable, setVariable] = useState(null);
+  const [climatology, setClimatology] = useState(null);
+
+  function selectVariable(event) {
+      setVariable(event.value);
+  }
+  
+  function selectClimatology(event) {
+      setClimatology(event.value);
+  }
+  
+  function dontSelectVariable(event){
+    //TODO: put something here. Ask Rod what.
+  }
```

Comment:
I originally envisioned cascading selection for model / variable / emissions scenario, similar to the way climate explorer does it, which we had many discussions about, and you wrote the simplify branch based on the needs I described at that time.

But eventually I decided that since this project features completely separate daily, monthly, and yearly indicators that would be graphed in different ways, I would not use a single "variable" selector, which scuttled the plan to use cascading selection. Instead I would put the yearly, monthly, and daily variables in different tabs with different selectors.

-------------------------

I *do* want to use cascading selection for Area Selection - we think we want to have people pick a major watershed, like "Fraser" or "Coastal," and use that to populate a dropdown with all the salmon species in that watershed, and have people pick a species, and then populate the next dropdown with the conservation units for that species. So I will talk to you about that when I get there!

Reply:
:+1: